node/components/network/src/gossip/tests/mod.rs

```diff
@@ -518,12 +519,16 @@ async fn test_batch_votes_propagation() {
         for (i, mut cfg) in cfgs.into_iter().enumerate() {
             cfg.rpc.push_batch_votes_rate = limiter::Rate::INF;
             cfg.validator_key = None;
-            let (node, runner) = testonly::Instance::new_from_config(testonly::InstanceConfig {
-                cfg: cfg.clone(),
-                block_store: store.blocks.clone(),
-                attestation: attestation::Controller::new(Some(setup.attester_keys[i].clone()))
-                    .into(),
-            });
+            let (node, runner) = testonly::Instance::new_from_config(
+                testonly::InstanceConfig {
+                    cfg: cfg.clone(),
+                    block_store: store.blocks.clone(),
+                    attestation: attestation::Controller::new(Some(setup.attester_keys[i].clone()))
+                        .into(),
+                },
+                |_| true,
```

Comment:
nit: annotate these lambda arguments, like: `/*filter_predicate=*/|_| true, /*selection_function=*/...` for readability.